recipe/meta.yaml

```diff
@@ -28,9 +28,9 @@ requirements:
     - python
     - cython
     - mumps-seq  # [not win]
-    - numpy 1.11
+    - numpy
     - scipy
-    - pytest
+    - pytest =4.6.4
```

Comment:
You probably need the pin in `run` as well.

Reply:
Though IDK why you need pytest in the env at run time.

Reply:
You can list a set of requirements in the test section if you want.

Reply:
Such that people can run `kwant.test()` and see if all linear algebra works for example.

Reply:
You need to put the pin in the run env too.

Reply:
here: https://github.com/conda-forge/kwant-feedstock/pull/71/files#diff-e178b687b10a71a3348107ae3154e44cR41